util/entropy_buffer_generator.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+"""Entropy Buffer Generator
+
+Creates an array of random bytes and stores it in a file.
+Typical usage:
+
+./entropy_buffer_generator.py -n 100
+
+This will create a file 'entropy_buffer.npy' consisting of 100 bytes.
+"""
+import argparse
+import random as rnd
```

Comment:
Inline with @cdgori 's comment in  #15144, I would just remove `as rnd` here. `random` is anyway just used once in this file.